src/Domain/TodoItems/AllTodoItemSpec.cs

```diff
@@ -1,8 +1,9 @@
-using Ardalis.Specification;
-using SSW.CleanArchitecture.Domain.TodoItems;
-
-namespace SSW.CleanArchitecture.Application.Features.TodoItems.Specifications;
-
-public sealed class AllTodoItemSpec : Specification<TodoItem>
-{
+using Ardalis.Specification;
+using SSW.CleanArchitecture.Domain.TodoItems;
+
+namespace SSW.CleanArchitecture.Application.Features.TodoItems.Specifications;
+
+// TODO DM: Consider removing this
```

Comment:
remove why?

Reply:
It's not actually encapsulating any query logic.

I'm also of the view that not all queries need to be implemented via specifications.  I would err on the siide of writing queries directly with EF, and refactoring to use specifications when the query needs to be shared/re-used.

Reply:
This has now been removed.